src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/publication/PublicationController.java

```diff
@@ -95,6 +97,9 @@ public class PublicationController {
     private final ExcelConversionService excelConversionService;
     private final PublicationServicesService publicationServicesService;
 
+    @Value("${ENV:local}")
```

Comment:
I don't believe this ENV variable is set anywhere and therefore will default to local for all env's.

I think we will either need to set for each environment in sds flux and the chart yaml, or alternatively change the name of this so it's specific to LCSU, something like 'LCSU_ENABLED', and then have that default to true and we set it to false in prod

Reply:
As discussed, it done.